From 26312a8774b5d113f6e7f904f7b6654449ab7b2e Mon Sep 17 00:00:00 2001
From: Daiki Ueno <dueno@redhat.com>
Date: Fri, 18 Aug 2017 11:54:05 +0200
Subject: [PATCH] common: Re-add placeholder definition of p11_debug

This was mistakenly removed in commit efe6dc56c.
Pointed by Lars Wendler in issue #97.
---
 common/debug.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/common/debug.h b/common/debug.h
index 4bf7e78..255c62c 100644
--- a/common/debug.h
+++ b/common/debug.h
@@ -144,6 +144,10 @@ void              p11_debug_precond             (const char *format,
 
 #else /* !defined (WITH_DEBUG) */
 
+#undef p11_debug
+#define p11_debug(format, ...) \
+	do {} while (false)
+
 #undef p11_debug_err
 #define p11_debug_err(errnum, format, ...) \
 	do {} while (false)